_sass/_va.scss

```diff
@@ -102,8 +100,8 @@ a {
 }
 
 a:visited {
-  color: #4c2c92
-  }
+  color: #4c2c92;
```

Comment:
Can we change this to `$color-visited` since it is already in `_variables.scss`?